documentation/docs/decentralized_identifiers/overview.md

```diff
@@ -9,68 +9,69 @@ keywords:
+W3C is a well-known standardization body that has defined how DIDs should look and work. This provides a basis for different technologies that implement the DID standard to achieve interoperability. A full list of all implementations can be found [here.](https://www.w3.org/TR/did-spec-registries/#did-methods) Please keep in mind that, unfortunately, most of these methods are outdated and not maintained.
 
 ## DID Documents
 
-The purpose of a DID is to help navigate to a DID Document, which is a document containing more information regarding the identity subject. This document contains data such as public keys, enabling the subject to prove ownership over their personal data, but also URIs that link to public information about the identity.
+The purpose of a DID is to help navigate to a DID Document, which is a document containing more information regarding the identity subject. This document contains data, such as public keys, enabling the subject to prove ownership over their personal data, but also URIs that link to public information about the identity.
 
-The identifier contains all information to resolve a DID, providing the latest DID Document. The first three characters `did` indicate that the DID standard from W3C must be used to resolve the identifier. It is followed by a unique method name, in our case `iota`, to indicate that the IOTA method is used. The IOTA method is a specific implementation that follows the following [method spec](../specs/did/iota_did_method_spec.md). This provides unique rules for the protocol to follow to result in the latest DID Document. In our case, it describes how DID Documents are uploaded and queried to and from the IOTA Tangle. Lastly, a DID contains a set of random characters that are unique per identity, this makes the identity unique and makes sure every identity resolves to a unique DID Document. 
+The identifier contains all information to resolve a DID, providing the latest DID Document. The first three characters `did` indicate that the DID standard from W3C must be used to resolve the identifier. It is followed by a unique method name, in our case `iota`, to indicate that the IOTA method is used. The IOTA method is a specific implementation that follows the following [method spec](../specs/did/iota_did_method_spec.md). This provides unique rules for the protocol to follow to result in the latest DID Document. In our case, it describes how DID Documents are uploaded and queried to and from the IOTA Tangle. Lastly, a DID contains a set of random characters that are unique per identity, making the identity unique and ensuring every identity resolves to a unique DID Document. 
 
-:::tip Requires basic knowledge of Asymmetric Encryption
+:::note Requires basic knowledge of Asymmetric Encryption
 
 The following and later sections require some basic knowledge of Asymmetric Encryption. Please read or view some materials on the subject before continuing.
```

Comment:
We decided against that. People can easily find some resources themselves. But if you find one that you guys consider very helpful, it would be great.

Reply:
I'd still alter or remove the second sentence. As is, it is a quite vague call to action ("go read something somewhere idk"), and also in the current phrasing it implies that the reader doesn't know basics of asymmetric encryption.

Also asymmetric encryption is just a term so it shouldn't be capitalized.